src/test/java/com/sap/oss/phosphor/fosstars/model/value/SecurityReviewsValueTest.java

```diff
@@ -25,4 +27,24 @@ public void testJsonSerialization() throws IOException {
     assertTrue(value.equals(clone) && clone.equals(value));
     assertEquals(value.hashCode(), clone.hashCode());
   }
+
+  @Test
+  public void testNoInfiniteRecursionLoopExists() throws IOException {
+    GitHubProject project = new GitHubProject("org", "test");
```

Comment:
`project` is not used anymore. The test case is a dup of the one above. Please remove it.